ross/tests/test_rotor_assembly.py

```diff
@@ -1557,27 +1557,34 @@ def rotor_6dof():
     return Rotor(shaft_elem, [disk0, disk1], [bearing0, bearing1])
 
 
-@pytest.mark.skip(
-    reason="Needs investigation. It fails depending on system. Most likely due to eig solution precision"
-)
+# @pytest.mark.skip(
+#   reason="Needs investigation. It fails depending on system. Most likely due to eig solution precision"
+# )
```

Comment:
Since we are going to use the test, I think we could just remove the skip instead of just commenting it.